src/rpm.rs

```diff
@@ -1721,4 +1721,277 @@ impl RPMBuilder {
 }
 
 #[cfg(test)]
-mod tests;
+mod tests {
+    use super::*;
+    use std::io;
+    use std::io::prelude::*;
+
+    #[test]
+    fn test_header() -> Result<(), Box<dyn std::error::Error>> {
+        let d = std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        let mut rpm_file_path = d.clone();
+        rpm_file_path.push("tests/assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm");
```

Comment:
you are referencing a file that does not exist. You've removed test_assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm but you did not add it to the tests directory